.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is being ignored. Ensure this is intentional, as it might contain important media files that need to be versioned. If these files are necessary for the project, consider removing this line.